api/client/samples/crop_models/brazil_soybeans.ipynb

```diff
@@ -57,7 +57,7 @@
      "name": "stdout",
```

Comment:
<p>The chart is conflicting what the description above this cell says? In the description syas <code>the crop-weighted NDVI series was built with weights normalized to 1.0</code>, therefore in the chart the blue line -- crop_weighted_value -- should have a value on y axis between 0 and 1? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/202/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>

Reply:
fixed

Reply:
looks good now